kodiak/config.py

```diff
@@ -72,6 +72,12 @@ class Merge(BaseModel):
     dont_wait_on_status_checks: List[str] = []
     # immediately update a PR whenever the target updates
     update_branch_immediately: bool = False
+    # ignore specific status checks that would normally prevent a PR from being
+    # updating. If for example the WIP status check was incomplete, we could
+    # specify "WIP" in this configuration to enable kodiak to update a branch
```

Comment:
Just to make sure we are on the same page: in our use case we dont use the WIP app from github marketplace, but the WIP prefix from `blacklist_title_regex` config